tests/unclassified/test_unclassified_tts.py

```diff
@@ -280,7 +276,8 @@ def test_fastspeech(self):
         callback = nemo.core.SimpleLossLoggerCallback(
             tensors=[loss_t], print_func=lambda x: logging.info(f'Train Loss: {str(x[0].item())}'),
         )
-        optimizer = neural_factory.get_trainer()
+        # Instantiate an optimizer to perform `train` action
+        optimizer = nemo.backends.pytorch.actions.PtActions()
         optimizer.train(
```

Comment:
Shouldn't this be nf.train()?

Reply:
I have copied that from one of the existing examples a long time ago. This is the call that I was consistently using along all tests/examples that I have refactored. 

So when  I was removing the local instance of NF I did the same here. At the end it is doing the same as self.nf.train(), i.e. creating action instance.

Reply:
But this is against the nemo design pattern.
If you think this is outside the scope of PR, that's fine, but we should make a note to change then back to self.nf...